4-projects/business_unit_1/nonprod/example_base_shared_vpc_project.tf

```diff
@@ -23,29 +23,13 @@ module "base_shared_vpc_project" {
   skip_gcloud_download        = var.skip_gcloud_download
   environment                 = local.env
   env_code                    = var.env_code
-  vpc_type                    = "base"
+  vpc_type                    = "private"
 
   # Metadata
-  project_prefix    = "prj-${local.business_code}-${var.env_code}-sample-${var.vpc_type}"
-  application_name  = "prj-${local.business_code}-sample-${var.vpc_type}-vpc"
+  project_prefix    = "prj-${local.business_code}-${var.env_code}-sample"
```

Comment:
Similar to previous comments I think we can do away with the local and env code variable

Reply:
I will take care of the local in a separate PR. env is now hardcoded. Tracked in #119